runtime/oti/stackwalk.h

```diff
@@ -338,12 +338,12 @@ extern "C" {
 #define JIT_RESOLVE_PARM(parmNumber) (walkState->walkedEntryLocalStorage->jitGlobalStorageBase[jitArgumentRegisterNumbers[(parmNumber) - 1]])
 #define J9SW_JIT_STACK_SLOTS_USED_BY_CALL 0x0
 #define J9SW_POTENTIAL_SAVED_REGISTERS 0x20
-#define J9SW_REGISTER_MAP_MASK 0xFFFFFFFF
+#define J9SW_REGISTER_MAP_MASK 0xFFFF0000
```

Comment:
This change makes `jitWalkRegisterMap` disregard lower 16 bits of `registerMap` for helper calls.
https://github.com/eclipse/openj9/blob/03d9188253d3cd031a35650740535c11db4c9582/runtime/codert_vm/jswalk.c#L701

Reply:
Yes, you're right, although I don't recall what I had in mind when I wrote this :-) .  I'll revert it to the way it was and then mask out the RSD field in the metadata to clear the bits I don't want (other architectures seem to do this using a local mask when building the metadata).